### PR TITLE
Added works and about myself section

### PR DESCRIPTION
**Pull request by Dadadon**
- Added works section
- Added about myself section
- Implemented the button interactions

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,31 +19,226 @@
...
     </section>
+
+    <div class="grid-container">
+        <section class="card1">
+            <section><img class="portfolio-image" src="src/Snapshoot Portfolio.png" alt=""></section>
+            <div class="p-layout">
+                <h1 class="portfolio-head">Tonic</h1>
+                <ul class="frame">
+                    <li>
+                        <h2 class="canopy">CANOPY</h2>
```
Comment (cwaku) [inline index.html]: - [ ] Good job with your project. I would have approved it right away but it's part of the main requirement to have the main container of this section with a `section` tag and each project preferably a `div` tag. Kindly reverse what you have done here and there should be nothing else to hold your project back from getting approved. Each project shouldn't be its own section.

**Review by cwaku — CHANGES_REQUESTED** [review]
Hi @Dadadon 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!
There are a few issues 🐛  that you still need to work 🛠️  on to go to the next project ⏭️  but you are almost there! 🏃🏾 

## Highlights
- [x] Good overall design 👍🏾 
- [x] Linters passed 💯 
- [x] Good commit history 🥇 
- [x] Meaning PR title and description 💯 
- [x] Correct use of Github flow 👍🏾 

## Required Changes ♻️
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips 🌟  that can improve your code quality 💯 and help you get a better job 💼.

I will guide 📔 you through fixing 🛠️ these bugs, follow my comments down below and make sure you don't miss any of them, in order to get your project approved ✅

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Dadadon) [reply index.html]: Okay thanks, i'll make the necessary changes

**Review by cwaku — APPROVED** [review]
Hi @Dadadon 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
 - [x] Good job overall following the template design 💯 
 - [x] Correct GitHub Flow 👍🏾 
 - [x] Good commit history 🥇 
 - [x] Good PR title/description 💯 
 - [x] All linters passed 🥇 
 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**